FlightMatcher/FlightMatcher/Location.swift

```diff
@@ -1,22 +1,22 @@
-//
-//  City.swift
-//  FlightMatcher
-//
-//  Created by Daria Korneichuk on 3/31/17.
-//  Copyright © 2017 Dmitry Smolyakov. All rights reserved.
-//
-
-import UIKit
-
-class Location {
-    let id: Int
-    let city: String
-    let country: String
-    
-    init(with id: Int, city: String, country: String) {
-        self.id = id
-        self.city = city
-        self.country = country
-    }
-
-}
+//
+//  Location.swift
+//  FlightMatcher
+//
+//  Created by Daria Korneichuk on 3/31/17.
+//  Copyright © 2017 Dmitry Smolyakov. All rights reserved.
+//
+
+import UIKit
+
+class Location {
+    let id: Int
+    let city: String
+    let country: String
+    
+    init(with id: Int, city: String, country: String) {

```

Comment:
Initializer and factory method calls should form a phrase that does not include the first argument e.g.
```swift
Color(red: 32, green: 64, blue: 128)
```
not
```swift
Color(havingRGBValuesRed: 32, green: 64, andBlue: 128)
```
In this case it would be better to make it Location(id: .....). The same applies to Request class.